src/univariate/discrete/betabinomial.jl

```diff
@@ -85,17 +85,17 @@ function pdf(d::BetaBinomial, k::Int)
     chooseinv = (n+1) * beta(k+1, n-k+1)
     numerator = beta(k + α, n - k + β)
     denominator = beta(α, β)
-    return (numerator / denominator) / chooseinv
+    return numerator / (denominator * chooseinv)
 end
 
 function pdf(d::BetaBinomial)
     n, α, β = d.n, d.α, d.β
     denominator = beta(α, β)
-    values = Array{Float64, 1}()
-    for k = 0:n
+    values = Array(Float64, n+1)
```

Comment:
This syntax is "deprecated" (according to the documentation it is, but Julia itself doesn't emit a deprecation warning for it). The preferred syntax is now `Array{T,N}(dims)`, where `T` is the type, `N` is the number of dimensions, and `dims` is the dimension specification. So you could write this as `Array{Float64,1}(n+1)`.